presto-product-tests/README.md

```diff
@@ -217,8 +230,8 @@ breakpoint is hit.
 ## Troubleshooting
 
 Use the ``docker-compose`` and ``docker`` utilities to control and troubleshoot
-containers. In the following examples ``<profile>`` is either ``singlenode``, ``multinode`` or
-``singlenode-kerberized``.
+containers. In the following examples ``<profile>`` is either ``singlenode``, ``multinode``, ``singlenode-kerberized`` or
```

Comment:
Do we need to list all the profiles once again here? Can't we just put the reference to the section above?